src/thx/LocalYearMonth.hx

```diff
@@ -0,0 +1,342 @@
+package thx;
+
+using thx.Ints;
+using thx.Strings;
+import thx.DateConst.*;
+
+/**
+`Date` represents a date (without time) between 5879611-07-12 and -5879611-07-13
+(the actual boundary values are platform specific and depend on the precision
+of the `Int` type).
+`Date` represents a moment in time with no time-offset information.
+*/
+abstract LocalYearMonth(Int) {
+/**
+Returns the system date/time relative to UTC.
+*/
+  public static function now() : LocalYearMonth
+    return fromDate(Date.now());
+
+/**
+Returns a LocalYearMonth instance from an `Int` value. The value is the number of days
```

Comment:
This also makes me wonder, does fromInt in LocalDate take into account the Gregorian reformation?